web/endpoint.go

```diff
@@ -1,19 +1,21 @@
-package yoti
+package web
 
 import (
 	"fmt"
 )
 
-func getProfileEndpoint(token, sdkID string) string {
+// GetProfileEndpoint gets the Profile Endpoint URI
+func GetProfileEndpoint(token, sdkID string) string {
 	return fmt.Sprintf("/profile/%s?appId=%s", token, sdkID)
 }
 
-func getAMLEndpoint(sdkID string) string {
+// GetAMLEndpoint gets the AML Endpoint URI
+func GetAMLEndpoint(sdkID string) string {
 	return fmt.Sprintf("/aml-check?appId=%s", sdkID)
 }
 
 // GetDynamicShareEndpoint gets the Dynamic Share Endpoint URI
-func getDynamicShareEndpoint(client clientInterface) (string, error) {
+func GetDynamicShareEndpoint(client ClientInterface) (string, error) {
```

Comment:
Perhaps we can move these into their own services/packages? (then we can keep them private within those packages)

Reply:
👍 moved in db15f25, will make that unexported in the next commit